doc/lspconfig.txt

```diff
@@ -354,8 +358,8 @@ for some project structures. Example (for Kotlin): >lua
     'build.gradle.kts', -- Gradle
   }
   root_dir = function(fname)
-    local primary = util.root_pattern(unpack(root_files))(fname)
-    local fallback = util.root_pattern(unpack(fallback_root_files))(fname)
+    local primary = vim.fs.dirname(vim.fs.find(root_files, { path = fname, upward = true })[1])
+    local fallback = vim.fs.dirname(vim.fs.find(fallback_root_files, { path = fname, upward = true })[1])
```

Comment:
is `vim.fs.root()` not relevant here? I know the configs are not using it for back-compat, but our docs could recommend stuff available in Nvim 0.10/0.11